### PR TITLE
Align 3.0.4 with 3.1.1

### PR DESCRIPTION
**Pull request by ralfhandl**
Editorial changes taken over from 3.1.1.

If we decide to reject one of these changes, we have to decide whether we want to transfer the 3.0.4 wording back to 3.1.1 via PR #3953

### CODE REVIEW

**Review by miqui — APPROVED** [review]
@ralfhandl approved, but not merge until your initial concern be addressed?

**Comment by ralfhandl** [comment]
> not merge until your initial concern be addressed?

Once we get a second approval this PR can be merged.

Goal is to avoid accidental differences between 3.0.4 and 3.1.1.

**Review by lornajane — APPROVED** [review]
Thanks @ralfhandl !

**Review by handrews — APPROVED** [review]

**Pull request merged**
